client/src/pages/Profile.jsx

```diff
@@ -1,14 +1,59 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
+import { getAttendingSessions } from '../utils/apiWrapper';
+import 'semantic-ui-css/semantic.min.css';
+import '../css/Profile.scss';
+import SessionSummary from '../components/SessionSummary';
+import ClassForm from '../components/ClassForm.jsx';
 import ClassCard from '../components/ClassCard';
 
 function Profile({ user }) {
+  const [sessions, setSessions] = useState([]);
+  const [classes, setClasses] = useState([]);
+
+  useEffect(() => {
+    const populateSessionsAndClasses = async () => {
+      console.log(user);
+      const resp = await getAttendingSessions(user._id);
+      if (!resp.error) {
+        console.log(resp.data.result);
+        setSessions(resp.data.result);
+      }
+      setClasses(user.classes);
+    };
+
+    populateSessionsAndClasses();
+  }, [user]);
+
   return (
     <>
-      <h1>
-        Profile Page for {user.firstName} {user.lastName}
-      </h1>
-      <ClassCard classCardText={'CS125'} />
+      <h2>Welcome to your Profile Page, {user.firstName}</h2>
+
+      <h1>My Sessions</h1>
+      <p></p>
+      {sessions.map((session, i) => (
+        <SessionSummary user={user} session={session} key={i} />
+      ))}
```

Comment:
Since merging the DetailsModal, you'll need to display this by using DetailsModal instead of SessionSummary. Please pull from main and change this to the following. See below in `Home.jsx`:  https://github.com/hack4impact-uiuc/academy-hw-study-buddies/blob/052c4e17dee3f76a3a8734ee805c76219546b854/client/src/pages/Home.jsx#L35-L38

When you click on a SessionSummary on the Profile Page, its details should pop up, just like they do on the Home Page.